lib/Backend/IRBuilderAsmJs.cpp

```diff
@@ -2674,7 +2674,14 @@ IRBuilderAsmJs::BuildInt3(Js::OpCodeAsmJs newOpcode, uint32 offset, Js::RegSlot
     case Js::OpCodeAsmJs::Imul_Int:
         instr = IR::Instr::New(Js::OpCode::InlineMathImul, dstOpnd, src1Opnd, src2Opnd, m_func);
         break;
-
+#if defined(_M_IX86) || defined(_M_X64)
+    case Js::OpCodeAsmJs::RotL_Int:
+        instr = IR::Instr::New(Js::OpCode::ROL, dstOpnd, src1Opnd, src2Opnd, m_func);
```

Comment:
we should have a RotL backend opcode, and in lower.cpp dispatch to appropriate machine dependent path where in LowerMDShared.cpp we simply change opcode (and we stub out ARM/ARM64, which will eventually need to be implemented.)